pkg/storage/postgres/postgres.go

```diff
@@ -431,25 +431,12 @@ func (p *Postgres) ReadAuthorizationModels(ctx context.Context, store string, op
 	return models, token, nil
 }
 
-// FindLatestAuthorizationModelID see [storage.AuthorizationModelReadBackend].FindLatestAuthorizationModelID.
-func (p *Postgres) FindLatestAuthorizationModelID(ctx context.Context, store string) (string, error) {
-	ctx, span := tracer.Start(ctx, "postgres.FindLatestAuthorizationModelID")
+// FindLatestAuthorizationModel see [storage.AuthorizationModelReadBackend].FindLatestAuthorizationModelID.
```

Comment:
This comment and the others in this PR need to update the `FindLatestAuthorizationModelID` referenced in this sentence.

Please do a project-wide find/replace on `FindLatestAuthorizationModelID`.